financial-templates-lib/test/price-feed/CryptoWatchPriceFeed.js

```diff
@@ -48,7 +48,7 @@ contract("CryptoWatchPriceFeed.js", function(accounts) {
     networker = new NetworkerMock();
     const dummyLogger = winston.createLogger({
       level: "info",
-      transports: []
```

Comment:
This change was unrelated to this PR but was added to prevent this transport from printing logs during tests. This now conforms to the rest of our testing Winston framework.